src/js/engagement_view/src/components/generators/generators.tsx

```diff
@@ -71,7 +71,7 @@ const MetricContainer = () => {
                         style={{ fontSize: "30px" }}
                         className={classes.metricsIcon}
                     />
-                    <h1>1</h1>
+                    <h1 className={classes.metricCount}>1</h1>
```

Comment:
For all of the fake values can we just put 0? Or N/A, since nothing is deployed.